backend/src/main/java/com/empire/Army.java

```diff
@@ -86,7 +100,7 @@ public double raze(World w, String order, Character leader, int inspires, boolea
 		for (int i = 0; i < razes; i++) {
 			Construction bestRaze = null;
 			for (Construction c : region.constructions) {
-				if (target.contains(c.type) && (!"temple".equals(c.type) || target.contains(c.religion.toString()))) {
+				if (target.contains(c.type.toString().toLowerCase()) && (c.type != Construction.Type.TEMPLE || target.contains(c.religion.toString()))) {
```

Comment:
This is not the time/place to start with this project but looking at this line, having to check the army's order string like this makes me think there should be a dedicated class for all those "unit orders".

Reply:
Yeah. Figuring out the data structure for orders is an interesting project!